lib/graphql/queries.js

```diff
@@ -700,6 +702,80 @@ const getCollectiveCoverQuery = gql`
   }
 `;
 
+const getCollectiveAndHostQuery = gql`
```

Comment:
Was that really necessary to introduce this new query ?

Reply:
I didn't want to overload a widespread query with the async operation of loading the collective's plan.
But I think I'm overcomplicating things, perhaps we should keep it simple and get back to it if it actually becomes a problem.